Voici mon résultat de votre calcul :

v= vitesse du soleil 230047.6424 m/s

c=299792458 m/s

In [1]:
%display latex
v=230047.6424;c=299792458
dt=sqrt(1/(1-(v^2/c^2)));n(dt)

1.00000029441800

In [2]:
(dt-1)

2.94418003976560e-7

Je suppose que dt-1=0.000000294180 est une unité ?

"C‘est l ‘ application de la relativité restreinte : la variation d’une unité de temps causée par
la vitesse du soleil." Voulez vous dire que le soleil ralenti la lumière, par unité temps /seconde ?

"Pour ma part j’en suis à la vitesse de la lumière sous forme d’un nombre complexe : 
( 299792458 + 7110718,049 i ) mètres / seconde"

Ca fait quelle vitesse en réelle, je pense que vous élevez au carré pour éviter le $i$ ?

(vous) :Voici mon calcul de la complexité de la vitesse de la lumière

J’utilise pi ( 3,14159265358 ) , le nombre d'or ( 1,618033989 ) 

les logarithmes sont des logarithmes népériens sauf indication contraire.

j'utilise x pour une multiplication / pour une division.

1,618033989 x 10 = 16,18033989
logarithme de logarithme de logarithme de 16,18033989 = 0,023536622
arc tangente 0,023536622 = 1°,348300138
logarithme décimal de ( 3,14159265358 exposant 2 ) = 0,994299745 (A)
0,994299745 (A) exposant 1,348300138 = 0,992321985
1,348300138 / 0,993231985 = 1°,358732507 (E) ( argument du nombre complexe )
299792458 x tangente 1°,358732507 (E) = 7110718,049
nombre complexe exprimant la vitesse de la lumière en m / s : 299792458 + 7110718,049 i

In [3]:
deg(x)=x*180/n(pi);deg(atan(0.023536622))

1.34830016759202

In [4]:
fd=x^2-x-1;solve(fd==0,x)

[x == -1/2*sqrt(5) + 1/2, x == 1/2*sqrt(5) + 1/2]

In [5]:
phi=n((1+sqrt(5))/2);phi

1.61803398874989

Pourquoi multipliez vous par 10 ?

In [6]:
phi*10

16.1803398874989

Log log log 16.1803398874989=0,023536622 je ne sais pas comment faire ?

In [7]:
deg(atan(0.023536622))

1.34830016759202

In [8]:
log(3.14159265358^2,10)

0.994299745385560

In [9]:
0.994299745385560^1.34830016759202

0.992321984765476

In [10]:
1.34830016759202/0.992321984765476

1.35873253670851

In [11]:
299792458/7110718.049

42.1606448088827

In [12]:
rad(x)=x*n(pi)/180;deg(1/42.1606448088827)

1.35898726816936

In [13]:
299792458/42.1606448088827

7.11071804899999e6

In [14]:
299792458+7110718.049*i

2.99792458000000e8 + 7.11071804900000e6*I

In [15]:
299792458+7110718.049*i*i

2.92681739951000e8

Célérité plus faible ? Me suis je trompé ?